debian/changelog.in

```diff
@@ -1,7 +1,13 @@
 wake (@VERSION@-1) unstable; urgency=medium
-
+ 
   * Auto-generated release package.
+ 
+ -- Abrar Quazi <abrar.quazi@sifive.com>  Tue, 21 Jan 2025 15:45:00 -0800
+
+wake (44.2.0-1) unstable; urgency=medium
```

Comment:
I think this is supposed to say 44.1.0-1 (because it is describing the last release I made not the release you are making now)